woocommerce-gutenberg-products-block.php

```diff
@@ -66,7 +66,7 @@ function woocommerce_blocks_admin_unsupported_wp_notice() {
 		if ( should_display_compatibility_notices() ) {
 			?>
 			<div class="notice notice-error">
-				<p><?php esc_html_e( 'The WooCommerce Blocks feature plugin requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying WooCommerce Blocks.', 'woo-gutenberg-products-block' ); ?></p>
+				<p><?php esc_html_e( 'The WooCommerce Blocks feature plugin requires a more recent version of WordPress and has been paused. Please update WordPress to continue enjoying WooCommerce Blocks.', 'woocommerce' ); ?></p>
```

Comment:
How come we are changing the text domains here?

Reply:
I forgot to restore this file! Fixed with d50c94fbe5b809456f52ee8679923e3ca2c07ec0